### PR TITLE
improvement(cluster.py): Supporting update_db_packages that are supplied as tar.gz

### PR DESCRIPTION
**Pull request by roydahan**
In case the update_db_packages url contains rpms that are tar.gz,
this command will untar them before trying to install them.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3649,7 +3649,11 @@ def update_scylla_packages(node, _queue):
             node.remoter.send_files(new_scylla_bin, '/tmp/scylla', verbose=True)
 
             self._wait_for_preinstalled_scylla(node)
+            logging.info("unzipping any tar.gz rpms")
+            node.remoter.run('tar -xvf /tmp/scylla/*.tar.gz -C /tmp/scylla/', ignore_status=True, verbose=True)
```
Comment (amoskong) [inline sdcm/cluster.py:3653]: 1. can we add an existing check before compressing?
    - test -e /tmp/scylla/*.tar.gz 
2. It requires the rpm packages in the top directory, currently we won't install the rpms in the sub-directory of /tmp/scylla/

File: sdcm/cluster.py [hunk 2]
```diff
@@ -3649,7 +3649,11 @@ def update_scylla_packages(node, _queue):
...
+
             # replace the packages
+            logging.info("Installing rpms")
             node.remoter.run('yum list installed | grep scylla')
             node.remoter.sudo('rpm -URvh --replacepkgs --replacefiles /tmp/scylla/*.rpm',
```
Comment (amoskong) [inline sdcm/cluster.py:3658]: Do we want to support to install the rpms in all sub-directory?

File: sdcm/cluster.py (hunk 1 above)
Comment (bentsi) [reply sdcm/cluster.py:3653]: better to check that the files exist before untaring

Comment (roydahan) [reply sdcm/cluster.py:3653]: no, it doesn't really matter like we don't check that the rpms are there.

**Pull request merged**